### PR TITLE
Ref

### PR DESCRIPTION
**Pull request by HubertKwiatkowski**
passing all the finished tests (text field test will be committed on a separate branch as a commit on it's own)

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lint
-yarn pretty
+# yarn lint
```
Comment (Sumick) [inline .husky/pre-commit:4]: **question:** Why commands are disabled?

File: coverage/clover.xml [hunk 2]
```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Sumick) [inline coverage/clover.xml:1]: **issue:** add `coverage` directory to `.gitignore`, and remove it from repo

**Review by Sumick — CHANGES_REQUESTED** [review]
We need to remove `coverage` dir from repo, and add it to `.gitignore`

File: .husky/pre-commit (hunk 1 above)
Comment (HubertKwiatkowski) [reply .husky/pre-commit:4]: oh, forgot about it - I had to do it when you asked me to push it a few days ago - couldn't do it with 'yarn lint' active

**Review by Sumick — APPROVED** [review]
Looks good now!

**Pull request merged**